tachyon/zk/plonk/examples/tracking_testing.md

```diff
@@ -0,0 +1,21 @@
+# Tracking Testing Types Across Example Circuits
+
+## Currently Tested Types
+
+| Circuit             | FloorPlanner       | PCS     | LS      | Fully Tested?      |
+| ------------------- | ------------------ | ------- | ------- | ------------------ |
+| SimpleCircuit       | SimpleFloorPlanner | SHPlonk | Halo2LS | :heavy_check_mark: |
+| SimpleCircuit       | V1FloorPlanner     | SHPlonk | Halo2LS | :heavy_check_mark: |
+| SimpleLookupCircuit | SimpleFloorPlanner | SHPlonk | Halo2LS | :heavy_check_mark: |
+| SimpleLookupCircuit | V1FloorPlanner     | SHPlonk | Halo2LS | :heavy_check_mark: |
+| ShuffleCircuit      | SimpleFloorPlanner | SHPlonk | Halo2LS | :heavy_check_mark: |
+| ShuffleCircuit      | V1FloorPlanner     | SHPlonk | Halo2LS | :heavy_check_mark: |
+| ShuffleCircuit      | SimpleFloorPlanner | GWC     | Halo2LS | :x:                |
+| ShuffleCircuit      | V1FloorPlanner     | GWC     | Halo2LS | :x:                |
```

Comment:
Why are they 'x' on `Fully Tested?`

Reply:
The original code for `ShuffleCircuit` with `GWC` did not have all the needed test data for certain tests. In the future, the data should be added and the `Fully tested?` column should be removed.